kratos/sources/communicator.cpp

```diff
@@ -93,6 +93,26 @@ int Communicator::TotalProcesses() const
     return mrDataCommunicator.Size();
 }
 
+Communicator::SizeType Communicator::GlobalNumberOfNodes() const
+{
+    return mrDataCommunicator.SumAll(mpLocalMesh->NumberOfNodes());
+}
+
+Communicator::SizeType Communicator::GlobalNumberOfElements() const
+{
+    return mrDataCommunicator.SumAll(mpLocalMesh->NumberOfElements());
+}
+
+Communicator::SizeType Communicator::GlobalNumberOfConditions() const
+{
+    return mrDataCommunicator.SumAll(mpLocalMesh->NumberOfConditions());
+}
+
+Communicator::SizeType Communicator::GlobalNumberOfMasterSlaveConstraints() const
+{
+    return mrDataCommunicator.SumAll(mpLocalMesh->NumberOfMasterSlaveConstraints());
```

Comment:
I am not sure if this will work. Constraints are not treated in the ParallelFillComm or in partitioning .. 
so if constraints are added through ModelPart they are not in mpLocalMesh.

Reply:
I see

Lets wait for feedback from others and then maybe remove it

Reply:
I reomoved it again

Reason is that we have a similar problem with the `Geometries`

Maybe we should add those functions to the `ModelPart` instead then?

Reply:
The thing is `ModelPart` never makes a distinction between global and local values/functions.
Adding these functions there will be odd.